### ISSUE TITLE
fadeDuration is not affected by Time.timeScale

### ISSUE THREAD

**Issue by muscly** [KybernetikGames/animancer#129]
Hi!
The playback speeds of animations are affected by Time.timeScale, but fade durations are not.
Setting AnimancerComponent.UpdateMode to Normal again didn't work.
And I also tried various FadeMode. 

I couldn't find any clue from the Animancer source code. It seemed to use FrameData.deltaTime.
I wonder if there is a way that fade durations are affected by Time.timeScale.
Thanks.

**Comment by KybernetikGames** [comment]
I just tested it in the Quick Play example using the `TimeScale` script and it seems to work properly (everything is affected by time scale in Normal mode).

I know there's a [Unity bug when you change to or from AnimatePhysics mode](https://kybernetik.com.au/animancer/docs/bugs/update-modes) so this could be something similar. Check if it works properly when you leave the Update Mode as Normal from the start.

**Comment by muscly** [comment]
It occurred without modifying the Update Mode property of my Animator component or AnimancerComponent.UpdateMode.
I'm using Unity 2021.1 on PC. 
I'll make a sample project where this issue and [this](https://github.com/KybernetikGames/animancer/issues/127) occur and send it you.

**Comment by muscly** [comment]
I sent you the sample project via email under the title of 'A Sample Project for Issue #127 and #129'.

**Comment by KybernetikGames** [comment]
This looks like a Unity bug which happens in Unity 2021 but not 2020 or older.

In `AnimancerPlayable.PrepareFrame` where it sets `DeltaTime = info.deltaTime;` just put this after it:

```
#if UNITY_2021_1_OR_NEWER
            DeltaTime *= Time.timeScale;
#endif
```

I'll submit a bug report for it.

**Comment by muscly** [comment]
Thank you. It fixed the problem.

**Comment by KybernetikGames** [comment]
Since this is a bug fix, I'll keep the issue open until the next Animancer version is released.

**Comment by KybernetikGames** [comment]
[Animancer v7.0](https://kybernetik.com.au/animancer/docs/changes/animancer-v7-0/) is now available with this fix in it.

**Comment by bysameng** [comment]
I believe this bug also still exists on 2020.3.17 LTS, not just 2021.

My transitions looked really janky in slow-motion. I enabled the timeScale fix for 2020.3, which then fixed it completely.

**Comment by KybernetikGames** [comment]
This issue is fixed in [Animancer v7.2](https://kybernetik.com.au/animancer/docs/changes/animancer-v7-2).
